lib/xeroizer/models/contact.rb

```diff
@@ -50,18 +50,20 @@ class Contact < Base
       string        :website # read only
       decimal       :discount # read only
       boolean       :has_attachments
+      string        :xero_network_key
 
       has_many  :addresses, :list_complete => true
       has_many  :phones, :list_complete => true
       has_many  :contact_groups, :list_complete => true
-      has_many  :contact_persons, :internal_name => :contact_people
+      has_many  :contact_persons, :internal_name => :contact_people, :list_complete => true
```

Comment:
OK, I should probably do a PR to disable hashrocket syntax

Reply:
Can you elaborate? I'm not sure what you mean.